tools/stoc/temp_analysis_indic.xml

```diff
@@ -0,0 +1,271 @@
+
+One tabular file with coordinates in latitude and longitude (one column for each) including also the site ID (the one used in the count data file) 
+
+|
+
+**Output**
+
+Gam method ::
+
+ - Two tabulars that details computed indicator per year and globaly and one optional and additional table with the smoothed data.
+
+ - One plot that show trends across years and one optional and additional plot with the smooth method.
+
+|
+
+GlmmTMB method ::
+
+ - Two tabulars that details computed indicator per year and globaly.
+
+ - A plot that show trends across years.
+
+
+|
+
+**Source**
+
+UnPublished script available at http://www.vigienature.fr/sites/vigienature/files/atoms/files/analysestoceps_0.zip
+and the first version written by Romain Lorrilliere
+
+Source informations considering the raw FunctExeTemporalAnalysisofIndicatorTrait.r script:
+This script compute the indicator csi, cti and ctri per year and site, wich also correspond to a community weighted mean (CWM), and analyse its temporal evolution trend and create graphical vizualisation.
+As indicator you can thus use also a trait.
+
+
+Script needs the followings inputs
+ - stoc or community data not filtered with at least 4 columns: year, site, species, and abundance (without 0, because this would biaised the calculation of the community weighted mean)
+ - species details file with at least 2 columns: the species name or species ID (found in the community data or in stoc data) and one or more traits (e.g. the ssi if you would like to comute the csi)
+ - specialization details file
+ - spatial coordinates data file with at least 3 columns: longitude, lattitude and site
+ - file that stocks functions : "FunctTrendSTOCGalaxy.r"
+ - optional : precomputed community file
+
+Arguments are :
+ - method : gam or glmmtmb
+ - plot_smooth : add a vizualisation when used with gam method
+ - ic : compute and show confidence interval in plots (TRUE / FALSE)
+
+
+
+How to execute, eg :
+ # all files are available in github repo
+ # Smooth vizu + no ic, method gam, no precomputed community file
+ $ Rscript FunctExeTemporalAnalysisofIndicatorTrait.r test-data/Datafilteredfortrendanalysis2.tabular tabSpecies.csv species_indicateur_fonctionnel.tabular coordCarreSTOCfaux.tabular "ssi" "csi" "gam" "" "idindicatortrait" TRUE FALSE FunctTrendSTOCGalaxy.r
+
+
+Outputs are created in an Output repo :
+
+Gam method gives :
+  - csi_gammCOMPLET_France.tabular
+  - csi_gammParannee_France.tabular
+  - figcsi_carre_France.png
+
+ if smooth_plot option is set to TRUE, adds : 
+  - csi_gammsmoothFrance.tabular
+  - figcsi_plotFrance.png
+
+
+Glmmtmb method gives :
+  - csi_glmmTMB_France.png
+  - ggdata_csiFrance.tabular
+  - GlmmTMB_coefficient_csiFrance.tabular
+
+
+
+
+R library needed
```

Comment:
do these things really belong into the tool help?

Reply:
you're right, thank you!!!!